### ISSUE TITLE
CXY-V6, linear advance 

### ISSUE THREAD

**Issue by codemunzlp** [rhapsodyv/Marlin#14]
The Linear Advance setting causes the Extruder Motor of my X5SA-400 with the CXY-V6 Board to stop mid print. It won't move until I turn the power off and back on. There is no clog, the E motor just stops working.
[Configuration &  Configuration_adv.zip](https://github.com/rhapsodyv/Marlin/files/5612507/Configuration.Configuration_adv.zip)

**Comment by codemunzlp** [comment]
1st Step: activate linear advance in marlin firmware
2nd Step: Print something for a few hours, the E motor will stop suddently

**Comment by codemunzlp** [comment]
It should be fixable using this:
![Bildschirmfoto vom 2020-11-29 18-43-59](https://user-images.githubusercontent.com/63969082/100549375-f1d71f80-3272-11eb-8d59-8016a64ed220.png)

**Comment by rhapsodyv** [comment]
Where is this image from?

**Comment by dandantsui** [comment]
Looks like it is from here

https://www.thingiverse.com/thing:4588538

**Comment by Eliminateur** [comment]
Indeed, you can't use linear advance with the steppers in stealthchop mode, that's set from factory, linear advance requires spreadcycle mode.
That image you pasted is not exactly the best, as that would change the mode permanently but you'll remain in standalone mode. 

for the best operation they need to be in UART mode so that marlin can change between modes as it wants(plus a lot of other advanced stuff, like running the extruder in spreadcycle alone for the torque)

this caveat(about the modes) should be added as a comment block on the file before linear advance, as right now it says nothing.

**Comment by corrado-c** [comment]
some time ago I made the change to activate the UART, but it didn't work for me,
Are there any news or better instructions?
![image](https://user-images.githubusercontent.com/22625950/150684803-1f3c698e-2bff-4a1a-984d-128f760d844c.png)
